winter_messaging_outbox_transacton/message_listener.py

```diff
@@ -1,15 +1,27 @@
 import json
+import logging
 from datetime import datetime
 from uuid import UUID
 
 from injector import inject
+from pika import BasicProperties
+from pika.adapters.blocking_connection import BlockingChannel
+from pika.spec import Basic
 from sqlalchemy.engine import Engine
-
 from winter.core.json import json_decode
+from winter.messaging import Event
 from winter.messaging import EventHandlerRegistry
 from winter.messaging.event_dispacher import EventDispatcher
+from winter_messaging_outbox_transacton.event_processing_logger import EventProcessingLogger
 from winter_messaging_outbox_transacton.inbox.inbox_message import InboxMessage
 from winter_messaging_outbox_transacton.inbox.inbox_message_dao import InboxMessageDAO
+from winter_messaging_outbox_transacton.middleware_registry import MiddlewareRegistry
+from winter_messaging_outbox_transacton.timeout_handler import TimeoutHandler
+
+logger = logging.getLogger('event_handling')
+
+EVENT_HANDLING_TIMEOUT = 15
+RETRIES_ON_TIMEOUT = 1
```

Comment:
From my POV it's better to pass these to constructor, to allow customization and avoid global variables.

Reply:
It's passed to method. 
```
timeout_handler.timeout(seconds=EVENT_HANDLING_TIMEOUT, retries=RETRIES_ON_TIMEOUT)
```

It's just a contact, not a global variable. Maybe Might be worth moving it to a separate file.